frontend/views/containers/contributions/IncomeDetails.vue

```diff
@@ -191,19 +191,18 @@ export default {
 
       try {
         const incomeDetailsType = this.form.incomeDetailsType
-        const groupProfileUpdate = await sbp('gi.contracts/group/groupProfileUpdate/create',
-          {
+        await sbp('gi.actions/group/groupProfileUpdate', {
+          contractID: this.$store.state.currentGroupId,
+          data: {
             incomeDetailsType,
             [incomeDetailsType]: normalizeCurrency(this.form.amount),
             paymentMethods
-          },
-          this.$store.state.currentGroupId
-        )
-        await sbp('backend/publishLogEntry', groupProfileUpdate)
+          }
+        })
         this.closeModal()
       } catch (e) {
         console.error('IncomeDetails submit() error:', e)
-        this.$refs.formMsg.danger(L('Failed to update income details. {reportError}', LError(e)))
+        this.$refs.formMsg.danger(e.message)
```

Comment:
Why no longer translate the displayed error message?

Reply:
This error message comes already translated within `e.message`, which comes from the lower-down `GIErrorUIRuntimeError` string, so there's no reason to build a super-long redundant sentence.

Reply:
Hmm. For this particular error it seems like it might be coming from vuelidate... probably worth opening a separate issue for this.

However for the other errors that might occur, such as anything related to sending the GIMessage itself, `e.message` will contain the translated string from `controller/actions/group.js`.